README.md

```diff
@@ -12,9 +12,10 @@ This repository contains the ZKsync Solidity compiler.
 ## System Requirements
 
 Supported platforms:
-- **Linux: x86_64**
-   * MUSL-based static builds do not depend on system libraries and can be run on any recent Linux distribution.
-- **MacOS 11+: x86_64, arm64 (Apple silicon)**
+- **Linux: x86_64, ARM64**
+   * [MUSL](https://musl.libc.org)-based static builds are currently supported, but may be deprecated in the future; they do not depend on system libraries and can be run on any recent Linux distribution.
+   * Conventional builds with GNU libraries are substantially faster but may have stricter compatibility requirements.

```

Comment:
I think we need to rewrite this a bit, because in our case GNU libc builds do not have any strict compatibility requirements - it's the same fully static binaries.

Maybe, something like that? What do you think?

- **Linux: x86_64, ARM64**
  > [MUSL](https://musl.libc.org/)-based static builds are currently supported but may be deprecated in the future. Users are encouraged to switch to GNU `libc` builds which offer the same compatibility, are substantially faster, and are recommended for future use.

Reply:
That's better, but I'd flip it all and only mention MUSL in the end as something we keep for compatibility. GNU shall be documented as the only option.

Users do not switch to anything explicitly. The Hardhat plugin does it for them.

Reply:
Also 1ea0a63 + bd52869